content/guides/understanding-inbox-messaging.md

```diff
@@ -0,0 +1,77 @@
+# Understanding Inbox Messaging
+
+---
+
+1. TOC
+{:toc}
+
+---
+
+## Preface
+
+BookingSync provides the ability to integrate with the Inbox on multiple different channels/platforms we support. This guide will present
+how to implement this integration.
+
+
+## Purpose
+
+The purpose of this feature is to provide a seamless integration between internal messaging systems in certain Channels and the BookingSync platform, for adminstrators to be able to handle incoming inquiries in a centralized location.
+
+
+## Entities
+
+- **Inbox Conversation**: Represents a thread/conversation and is scoped under a Channel, an Account, and a Booking.
+<br>
+- **Inbox Participant**: Represents a participant in the thread and is scoped under an Inbox Conversation. There are two types of participants
+  - Client: Represents a Traveller, managed by the Channel entirely.
```

Comment:
`managed by the Channel entirely` - only if conversation comes from channel. 
If conversation is for a booking where channel like Airbnb or Bcom is not the source, it is us (if property manager starts conversation via Admin UI) or the partner (if API application is handling some specific/private inbox integration)